### PR TITLE
Support 1.3

### PR DESCRIPTION
**Pull request by Caaz**
Adds 1.3 support, My first attempt at modding.

What I did do
- Make this mod work in 1.3

What I didn't do
- Consider support for past versions using the fancy mod folder structure. 
- Verify every single feature works.

Things to think about
- `raidLootValueFromPointsCurve` is new, per 1.3. I set it to the other raid value, hopefully that'll work
- `CanReach` seems to have added a break fences option along side breaking doors. For this update, I assume if it can break a door, it can break a fence.
- Wound Rendering in general seems to have changed in 1.3, I don't know the specifics and mostly removed references to get it working. Might be worth looking into.
- `PawnGenerationRequest` is a hell of a function and I had to go and comment some stuff to figure out what changed, it's a bit messy in there, and could be cleaned up.
- I removed some extraneous dll build folders from visual studio
- Changed how libraries are imported, may be a problem if I didn't commit them right.

In general, this is my first experience coding on Windows. Just, bruh.

### CODE REVIEW

File: Defs/FactionDefs/Factions_Zombiefied.xml [hunk 1]
```diff
@@ -13,20 +13,17 @@
...
-    <startingGoodwill>
-      <min>-100</min>
-      <max>-100</max>
-    </startingGoodwill>
-    <mustStartOneEnemy>true</mustStartOneEnemy>
-    <naturalColonyGoodwill>
-      <min>-100</min>
-      <max>-100</max>
-    </naturalColonyGoodwill>
+		<permanentEnemy>true</permanentEnemy>
```
Comment (Zombiefied7) [inline Defs/FactionDefs/Factions_Zombiefied.xml:26]: Is there a reason for this line <permanentEnemy>? Might break interactions with some factions

**Comment by Zombiefied7** [comment]
Im confused that this would work. I heard that a lot of the pawn rendering was changed

File: Defs/FactionDefs/Factions_Zombiefied.xml (hunk 1 above)
Comment (Caaz) [reply Defs/FactionDefs/Factions_Zombiefied.xml:26]: yeah, `naturalColonyGoodwill` was removed, according to some other modders that I was talking to there were a few other options, including one that's "enemy to all factions except ..." which takes a list. Not sure how factions are really used in this mod.

**Comment by Caaz** [comment]
> Im confused that this would work. I heard that a lot of the pawn rendering was changed

just seemed like some rotation things were added, worth testing it more than I have, I only really got to test a tiny bit last night, and I've got work for a while so I won't be on top of it today. I think where rendering changed the most was the wound stuff.

**Review by Zombiefied7 — APPROVED** [review]
Zombie wounds dont work anymore but its good enough

**Pull request merged**